users/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from users.managers import UserManager
+
+
+class User(AbstractUser):
```

Comment:
why you inherited from this model? and why not from AbstractBaseUser?

Reply:
As I understand, AbstractUser is default Django's built-in user model, AbstractBaseUser has only authentication mechanism. Using AbstractUser allows me to use or override all built-in user's functionalities (and requires "switching off" username field)